Jobs/src/Job.IdentityDeletion/Program.cs

```diff
@@ -97,6 +98,11 @@ static IHostBuilder CreateHostBuilder(string[] args)
                 .AddModule<SynchronizationModule>(configuration)
                 .AddModule<TokensModule>(configuration);
 
+            services.AddMediatR(c => c
+                .RegisterServicesFromAssemblyContaining<CancelStaleIdentityDeletionProcessesCommand>());
+
+            services.AddModule<DevicesModule>(configuration);
```

Comment:
Can be removed. The DevicesModule is already added in line 92.

Reply:
Changes made.